CodeConverter/CSharpToVB/ExpressionVisitor.vb

```diff
@@ -1250,7 +1248,7 @@ Namespace CSharpToVBCodeConverter.DestVisualBasic
                                                     )
                     Return binaryExpressionSyntax3
                 Catch ex As InsufficientExecutionStackException
-                    MsgBox(ex.Message, MsgBoxStyle.Critical, "Stack Overflow")
+                    _reportException?.Invoke(ex)
```

Comment:
ℹ It is possible to write tests for this behavior now that the control is inverted rather than hardcoding a MsgBox call.

Reply:
I like it, thanks. I think I fixed what causes you to get here which is always very long string concatenations and I handle that in the code so it doesn't happen.